src/components/VmStatus/strings.js

```diff
@@ -0,0 +1,25 @@
+export const RUNNING = 'Running';
+export const ERROR = 'Error';
+export const VM_ERROR = 'VM Error';
+export const POD_ERROR = 'Pod error';
+export const IMPORTING_ERROR = 'Import error';
+export const PENDING = 'Pending';
+export const STARTING = 'Starting';
+export const IMPORTING = 'Importing';
+export const IMPORTING_VMWARE = 'Importing (VMware)';
+export const IMPORTING_PENDING_VMWARE = 'Import Pending (VMware)';
+export const IMPORTING_ERROR_VMWARE = 'Import Error (VMware)';
+export const VIEW_POD_EVENTS = 'View pod events';
+export const VIEW_POD_DETAILS = 'View pod details';
+export const VIEW_VM_EVENTS = 'View VM events';
+export const UNKNOWN = 'Unknown';
+export const OFF = 'Off';
+export const MIGRATING = 'Migration';
+export const IMPORTING_VMWARE_MESSAGE =
+  'The virtual machine is being imported from VMWare. Disks will be converted to the libvirt format.';
+export const IMPORTING_ERROR_VMWARE_MESSAGE = 'The virtual machine could not be imported from VMWare.';
+export const IMPORTING_MESSAGE = 'The virtual machine is being imported. Disks will be copied from source image.';
```

Comment:
If the disks are being copied as part of the import process, change to "Disks are being copied from the source image."